### PR TITLE
Refactor MatrixWorkspaceTest::createImage(...).

### PR DESCRIPTION
**Pull request by quantumsteve**
Description of work.

This just cleans up a helper function for `MatrixWorkspaceTest`. It can be written cleaner.

**To test:**

This addresses an issue found while reviewing PR #15935.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/API/test/MatrixWorkspaceTest.h [hunk 1]
```diff
@@ -1350,16 +1355,14 @@ class MatrixWorkspaceTest : public CxxTest::TestSuite {
 
 private:
   Mantid::API::MantidImage_sptr createImage(size_t width, size_t height) {
```
Comment (ElliotAOram) [inline Framework/API/test/MatrixWorkspaceTest.h]: @quantumsteve 
**Very minor** - Would it make sense to have `width` and `height` as `const`?
e.g.
`Mantid::API::MantidImage_sptr createImage(const size_t width, const size_t height) {...}`

Comment (quantumsteve) [inline Framework/API/test/MatrixWorkspaceTest.h]: Yes, that does better clarify intent.

**Comment by ElliotAOram** [comment]
After the amendment code looks good. It's good to see a lot of the use of `new` being replaced with shared pointers. :shipit:

**Comment by FedeMPouzols** [comment]
Great! There's quite a few ugly points around the "Mantid Images". This is a nice improvement.

**Pull request merged**